src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * duke.Ui represents the class that deals with interactions with the user
+ */
+public class Ui {
+    private final Scanner scanner = new Scanner(System.in);
+
+    /**
+     * Prints the duke.Response to the console.
+     * @param response The duke.Response to be printed.
+     */
+    public void printResponse(Response response) {
+        System.out.println(response.toString());
+    }

```

Comment:
Is this method necessary? I think it would be okay to directly use the System.out.println method instead of having this method.

Reply:
Yes it is, to respect abstraction.